src/BSON/Binary.c

```diff
@@ -210,14 +207,14 @@ static PHP_METHOD(Binary, jsonSerialize)
 */
 static PHP_METHOD(Binary, serialize)
 {
-	php_phongo_binary_t      *intern;
+	php_phongo_binary_t* intern;
 #if PHP_VERSION_ID >= 70000
-	zval                      retval;
+	zval retval;
 #else
-	zval                     *retval;
+	zval* retval;
 #endif
```

Comment:
Should we make this into a ``#define ZVAL_RETVAL_TYPE`` so we don't have the if/else/endif for these everywhere?

Reply:
Works for me. You can add that to `phongo_compat.h`. In that case, I'd suggest ordering that commit before the one that changes formatting with the first `clang-format` invocation.

Reply:
Oh, I was hoping the changes introduced by `clang-format` were all isolated in a single commit (separate from manual changes like you adding line breaks). That would allow you to throw out that commit and regenerate as needed. If not, don't worry about the ordering.

Reply:
I did end up fixing this btw, including right order.